client/src/Components/shared/TopNavBar/container.js

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import "./topnav.css";
+
+class TopNav extends React.Component {
+  render() {
+    return (
+      <div className="top-nav">
+        <div className="home-menu">
+          <img src={require("./icons/menu.svg")} alt="" />
+          <p>Home</p>
+        </div>
+        <div className="logo">
+          <p>Logo Goes Here</p>
+        </div>
+        <div className="notify-help">
+          <img
+            src={require("./icons/notification-bell.svg")}
+            alt="Notifications"
+          />
+          <img src={require("./icons/help-button.svg")} alt="Help" />
```

Comment:
I have to wonder how well these requires in the code will scale. It seems like a lot of effort to type it out, and what if you want to conveniently change your icons directory? Let's see an example function for this common `require`:
```javascript
class TopNav extends React.Component {
  requireSvg = name => require(`./icons/${name}.svg`);
  // in your render's return...
    <img src={this.requireSvg("menu")} alt="descriptive text" />
}
```

Reply:
Yeah. This will probably need to be removed. This isn't what the container is for.

Reply:
@mglittleton

Reply:
@BryceEvans

Reply:
This is mine. I'm not sure I understand what you mean by "this isn't what the container is for."

Reply:
@jessharrison83 Did our convo resolve this question? I was just seeing it.

Reply:
Yes it did. I’ll fix this tonight.